frontend/src/validators/ActiveEngagements.js

```diff
@@ -14,7 +15,8 @@ export default {
     return Validate.activeEngagements(employee, validity)
   },
 
-  getMessage (value, args, data) {
-    return data
+  getMessage (field, args, key) {
```

Comment:
This method is duplicated elsewhere; could we consolidate them into e.g. `i18n.js`?